pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -301,6 +301,103 @@ type MonitoringSpec struct {
 	// +kubebuilder:validation:Maximum=65535
 	// +optional
 	ExternalMgrPrometheusPort uint16 `json:"externalMgrPrometheusPort,omitempty"`
+	// CustomizedPrometheus points to a customized Ceph prometheus alerts
+	// +optional
+	// +nullable
+	CustomizedPrometheus *PrometheusRuleCustomized `json:"prometheusRule,omitempty"`
+}
+
+//PrometheusRuleCustomized keep customized alerts
+type PrometheusRuleCustomized struct {
```

Comment:
Will we ever have anything other than the alerts? This extra level of indirection seems unnecessary

Reply:
I think it can help us in the future (or event in this pr) if we would like to add "labels" section for example, as described in this [comment](https://github.com/rook/rook/pull/9503#discussion_r777610436).

If you don't think it's necessary, I can remove this extra level.

Reply:
It doesn't seem necessary, as commented above about labels.